examples/allocator-service/allocator-ingress.yaml

```diff
@@ -0,0 +1,16 @@
+# Create a single service Ingress resource
```

Comment:
I thought about using only a Load Balancer, but went with the Ingress because it seems easier to use TLS.  It also seems easier to piont a subdomain at the ip, and allows for more flexibility in the future.
I think the focus should be on how to safely allocate a gameserver from a fleet.  I'll try and edit out as much complexity as possible.

Reply:
I was thinking about this more - and I'm going to come back to this.

I feel strongly we should remove the Ingress piece here, it's too complicated, could intimidate new users, and doesn't add anything to the example.

I feel like this example is meant to show how to do a fleet allocation through the Kubernetes API. The rest is just implementation details - so the simpler the service the better. So remove Ingress, and remove the SSL. (I could even advocate for removal of the token auth -- but would leave a note about how this service is totally open, and point to article on locking down services).

We could definitely add to the "Next Steps" section, a link to existing documentation on how to setup ingress and/or a ssl cert, but I don't think we need this here.

In practice I feel like a user is going to integrate with a matchmaker, and not setup a direct API call like we have here anyway. And setting up a matchmaker is outside the scope of this project.